pkg/install/generate.go

```diff
@@ -294,6 +309,12 @@ func InstallerPodSpec(
 	}
 	cliImage := *cd.Status.CLIImage
 
+	hiveArgs := []string{fmt.Sprintf("/usr/bin/hiveutil install-manager --work-dir /output --log-level debug %s %s", cd.Namespace, provisionName)}
+	if cd.Spec.Platform.VSphere != nil {
+		// Add vsphere certificates to CA trust.
+		hiveArgs = []string{fmt.Sprintf("cp -v /vsphere/.cacert /etc/pki/ca-trust/source/anchors/ && update-ca-trust && /usr/bin/hiveutil install-manager --work-dir /output --log-level debug %s %s", cd.Namespace, provisionName)}
```

Comment:
It can be named anything, `.cacert` is just something I went with. `GOVC_TLS_CA_CERTS` accepts `:` delimited files but I join them into one in hiveutil.